controllers/volumereplicationgroup_controller.go

```diff
@@ -1477,6 +1477,12 @@ func (v *VRGInstance) PVUploadToObjectStore(pvc *corev1.PersistentVolumeClaim,
 			}
 
 			err = v.reconciler.PVUploader.UploadPV(storer, v, s3ProfileName, pvc)
+			if err != nil {
```

Comment:
I understand that this is a hack for now, but I just want to note that UploadPV can return an error that is not related to s3Store, and the subsequent PVs would be punished for it (for a few seconds or a few minutes at the most).  We should make sure that this hack is reworked for 4.11.